### ISSUE TITLE
New orders and customers are not saved

### ISSUE THREAD

**Issue by MartijnBraam** [magento/magento2#4431]
## Steps to reproduce
1. Upgrade to magento newer than 2.0.2
2. Order anything
## Expected result
1. New customer created in magento
2. New order created in magento
3. Customer emailed
4. Payment procedure
## Actual result

Customer gets an email with a new order number and the whole payment procedure is successful. There is nothing logged about the customer or order at all but the customer gets an order number in their email that is neatly sequential but just doesnt exist.

**Comment by maksek** [comment]
@MartijnBraam, there is not enough data in steps to reproduce. What is payment method used? What is configuration of orders in store? 
Did you check admin orders? Did you check logs if there are any info with errors? Did you try use developer mode for testing purpose?

**Comment by kgvconsulting** [comment]
@MartijnBraam 
@maksek 
@MomotenkoNatalia 

After upgrading to v 2.0.6 the same result with us too, the new orders does not show in the Sales/Orders menu of backend admin side. The customer get email confirmation and the new orders are shown ONLY on Dashboard, under latest order, but there is a space only for 5 new orders so if you go over that number you miss orders. It is plausible to open the new order from the dashboard, and do changes but you cant find, search or see the new order from Sales/Order menu see following screenshots

This is the Sales/Order table from backend Admin showing the test orders before we upgraded 
![screenshot 2016-06-10 10 55 03](https://cloud.githubusercontent.com/assets/11282348/15968736/045a4a4a-2efb-11e6-893c-b3842383ea74.png)

This is the test orders placed after the upgrade they shown on the Dashboard, but dont display at all in Orders

![screenshot 2016-06-10 10 55 24](https://cloud.githubusercontent.com/assets/11282348/15968834/6617c064-2efb-11e6-8cdc-9d7834a9c7de.png)

If I click on the latest order on the Dashboard I can open the order

![screenshot 2016-06-10 10 55 43](https://cloud.githubusercontent.com/assets/11282348/15969271/53755514-2efd-11e6-8809-5873ddb229de.png)

If I search the order # in this case 000000029 in Sales/Order table the order don't display at all

![screenshot 2016-06-10 10 57 00](https://cloud.githubusercontent.com/assets/11282348/15969286/636d6cb8-2efd-11e6-9004-c30e14b90a4b.png)

If I login as a customer under customer order dashboard all is displayed correctly

![screenshot 2016-06-10 10 59 02](https://cloud.githubusercontent.com/assets/11282348/15969081/538ab4be-2efc-11e6-9dce-8e6ec9614666.png)

This is the recorded transactions under admin backend menu - Sales/Transactions

![screenshot 2016-06-10 12 18 44](https://cloud.githubusercontent.com/assets/11282348/15971042/ac629fbc-2f05-11e6-9c54-283ad7578eb8.png)

As you see all transactions - orders after May 27th 2016 and the upgrade to v 2.0.6 are no longer recorded under admin backend - Sales/Orders nor under admin backend Customers/AllCustomers/(select any of the customers from the tabel)/Orders menus

So in conclusion the admin backend don't record save the new orders, the new orders don't record and display under admin backend Sales/Orders menu and Customers/All Customers/Selected customer/Orders menue

**Comment by ghost** [comment]
I am seeing this issue in 2.1 CE as well. Any updates on this?

Edit: The orders appear right away on the dashboard under Last Orders but do not appear under the orders page. After about 10-15 minutes the orders will appear under the orders page but with an incorrect Purchase Date. The incorrect time issue seems to be the same as #5651.

**Comment by kgvconsulting** [comment]
No. The issue is that every new Magento version overnight and change the database tables. Whoever made those upgrading don't take in consideration the database tables. I migrated to custom shopping cart application don't have time to waist with fixing Magento after every upgrade

On Mon, Aug 8, 2016 at 1:51 PM -0400, "crtsl" notifications@github.com wrote:

I am seeing this issue in 2.1 CE as well. Any updates on this?

**Comment by salelsol** [comment]
- Same issue in 2.1.1, i can´t show up newest order in backend Sales, just in Dashboard.

**Comment by aelakkad** [comment]
any update on this issue ? i face same issue , orders not appearing in Sales/orders but appears only on dashboard latest order !? i am using Magento ver. 2.1.3 ?

**Comment by RomanKrut** [comment]
@MartijnBraam Hello, i tried to reproduce your issue and i can`t do it, can you check your problem on the latest version of Magento 2?

**Comment by korostii** [comment]
Hi @krutkoroma79911, Hi @veloraven,
1) As far as I know, *2.1.7* is the latest stable release, is it not?
2) Is there a way to get 2.2.0 CE other than by cloning repository?
3) Are you really suggesting to use `develop` branch in production in order resolve various issues including this one? Is that official?

**Comment by MartijnBraam** [comment]
@krutkoroma79911 I cannot check the problem since I've removed magento 2 a year ago

**Comment by veloraven** [comment]
We were not able to reproduce this issue after upgrade to ver. 2.1.7 as well as on develop branch.
If the issue still reproduces on some of the latest versions (2.1.7, 2.0.15, develop) please provide exact steps for it and exact affected versions. 
Otherwise the issue will be closed.

**Comment by martinmarsian** [comment]
Sorry, just today when testing Magento CE 2.1.7 order processing I just stumbled too over this behaviour. I used Paypal testing accounts.
1. Order can be placed and paid
2. Orders show in Dashboard
3. Orders do not show in Sales->Orders
4. Customer receives email and order no.
5. You can open the order from dashboard, create invoices etc.
6. When going back to dashboard "Something went wrong" message appears three times.

Magento 2.17 is in developer mode. No Caches. SSL is not running.

I don't know what you mean with exact steps to reproduce. Just placing an order, paying with Paypal, getting the emails expected, but order is not in sales->orders where they should be visible.

This is also kind of urgent, as we want to go into production soon.

**Comment by RomanKrut** [comment]
@joergmaertin Please, tell us next information: 1) Did you update your Magento 2 or it was fresh instance? 
2) If you use updated version, tell us which payment method do you use if no create new github issue with your steps and information about it.

**Comment by martinmarsian** [comment]
@krutkoroma79911 
1) it is an update to 2.1.4, 2.1.4 was a fresh install, then updated to 2.1.7
2) I did not test checkout on 2.1.4, so I can't tell if the problem was already existent before
3) To test I used Paypal Plus, a solution of Paypal for Germany, from iways, https://github.com/i-ways/magento2-paypal-plus.
I did not test with SSL, as the staging site had no SSL. Just tomorrow I get SSL for the staging site, so I will do additional tests with SSL on.

**Comment by kgvconsulting** [comment]
Hello,




I stop using Magento after the reported by me issue occurred. It is obvious that with every new update Magento team litteraly ignore the previous versions of the application and that create almost 70% of all issues reported. So you fix or make better one thing and in the process break something that was working just fine.




So there are two acceptable alternatives for Magento users, either to stick with one version and never upgrade it, or to go through the unnecessary waist of time to try fix bugs created by your new core updates. 




So I choose the third option to quit and stop using Magento for good!




Don't bother to ask me more questions, never will go back to this platform. 




Regards




Krasimir Vatchinsky


President




KGV Consulting Corp


405 RXR Plaza Suite 405


Uniondale NY 11556




Tel. +1 800 824-2830


email: kvatchinsky@kgvconsultingcorp.com




www.kgvconsulting.com




Sent from Outlook Mobile







On Fri, Jun 30, 2017 at 9:50 AM -0400, "Roman Krut" <notifications@github.com> wrote:












@joergmaertin Please, tell us next information: 1) Did you update your Magento 2 or it was fresh instance?

2) If you use updated version, tell us which payment method do you use if no create new github issue with your steps and information about it.

**Comment by veloraven** [comment]
I'm closing this issue as neither reporter nor other commenters can provide us with information enough to reproduce this issue.
@joergmaertin  as I can see you mention external module. Please contact this module's developer with your problem.
If this issue still reproducible with some built-in payment method please feel free to create the new one: format new issue according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines): with steps to reproduce, actual result and expected result and specify Magento version.

**Comment by martinmarsian** [comment]
Well, what can I say, this issue is closed but I opened it again #10128.
This time I included debug.log and system.log with information taken at the time of the test order.
This time I used internal payment method Check/Money order, with same result.
SSL was active.

I hope that helps. I am not that frustrated as one of the earlier commenters. But this isn't actually a new issue, the first post was from May last year and nothing really happened about it. So I hope the information given will help to resolve that issue, as it is somewhat crucial.
